docs/configuration.md

```diff
@@ -141,26 +141,15 @@ _note: `footer_content` is deprecated, but still supported. For a better experie
 
 ## Color scheme
 
-```yaml
-# Color scheme supports "light" (default) and "dark"
-color_scheme: dark
-```
+- `color_scheme`: Color scheme currently supports `nil` (default), `"auto"`, `"light"`, `"dark"` or a custom scheme that you define.
+- `enable_local_storage_theme`: Enable the local storage of the current theme
+- `enable_switch_theme_button`: Show the switch theme button on the aux-nav-list.
+- `switch_theme_available_color_scheme`: Set the available theme for the switch button, the default is `["auto", "light", "dark"]`. You can add a custom scheme or remove a scheme.
+- `switch_theme_available_timeout_fart`: If the remote server is slow to respond and FART (Flash of inAccurate coloR Theme) occur, to avoid them it is possible to increase the time (the default is `100`) in which two themes are loaded correctly and then wait longer for the theme change. 
+
+
 
-<button class="btn js-toggle-dark-mode">Preview dark color scheme</button>
-
-<script>
-const toggleDarkMode = document.querySelector('.js-toggle-dark-mode');
-
-jtd.addEvent(toggleDarkMode, 'click', function(){
-  if (jtd.getTheme() === 'dark') {
-    jtd.setTheme('light');
-    toggleDarkMode.textContent = 'Preview dark color scheme';
-  } else {
-    jtd.setTheme('dark');
-    toggleDarkMode.textContent = 'Return to the light side';
-  }
-});
-</script>
+_note: this switch does not save to cache or local storage the last choice made by the user. 
```

Comment:
Can we either make this a callout, or use the proper markdown?

Additionally, copy adjustment:
```suggestion
*Note:* `enable_local_storage_theme` *must* be enabled for changes to persist across pages and sessions.
```

Reply:
I had copied the _note syntax which was already present, but it did not work...